### PR TITLE
Changed to use the vendor version of tinymce which is an older version

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@stulees please review.

Presentation here:
http://rva.risevision.com/#/PRESENTATION_MANAGE/id=4b198c61-bbb7-4f75-9a99-2f079444070e?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
@stulees please note that the build has run successfully on cci but github is not updating. cheers

**Comment by stulees** [comment]
@rodrigopavezi That vendor version has a version of `4.3.13 (2016-06-08)` - https://goo.gl/o1ZWCb . Is it possible we could point to that specific version via CDN? Or via npm/bower? I'm weary of relying on that dist/ folder because it can very easily get overwritten in our build and also we have made a point of not committing dist/ folders anymore in our widgets.

**Comment by rodrigopavezi** [comment]
@stulees thanks, made changes. please review. cheers

File: bower.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.4.0",
     "widget-common": "https://github.com/Rise-Vision/widget-common.git#3.2.2",
     "widget-settings-ui-components": "https://github.com/Rise-Vision/widget-settings-ui-components.git#1.2.12",
-    "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git#0.4.3"
+    "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git#0.4.3",
+    "tinymce": "4.5.2"
```
Comment (stulees) [inline bower.json]: @rodrigopavezi I think the version used previously in the dist/vendor was `4.3.13`, any reason for using 4.5.2?

File: src/settings.html [hunk 2]
```diff
@@ -133,6 +128,12 @@ <h2 class="modal-title">{{"font-setting.custom-font" | translate}}</h2>
   </script>
 
   <!-- build:js js/settings.min.js -->
+  <script src="components/tinymce/tinymce.js"></script>
```
Comment (stulees) [inline src/settings.html]: @rodrigopavezi Can you confirm everything works as normal now that you're not loading tinymce code in the `<head>`?

File: bower.json (hunk 1 above)
Comment (rodrigopavezi) [inline bower.json]: @stulees that was the latest version that the bug does not exist. cheers

Comment (stulees) [inline bower.json]: @rodrigopavezi Okay cheers

File: src/settings.html (hunk 2 above)
Comment (rodrigopavezi) [inline src/settings.html]: @stulees yeap, everything looks to be working fine. cheers

**Comment by stulees** [comment]
@rodrigopavezi One comment regarding now loading tinymce in <body>, otherwise 👍

File: src/settings.html (hunk 2 above)
Comment (stulees) [inline src/settings.html]: @rodrigopavezi Nice

**Pull request merged**